CrossWordDB.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2022-10-30T09:35:32.423Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36" etag="MV5hUkhRPX3GHER6isz4" version="20.5.1" type="device"><diagram id="C5RBs43oDa-KdzZeNtuy" name="Page-1">7ZltT9swEIB/TaXtA1OT0BY+ksAYo6BBt7F9NImbeDi+znHoy6/fObGbhqCuILJqUqQK6vPZte+5FzvpeUG6OJdkllxBRHnP7UeLnnfac12n7x3hPy1ZlpLB4agUxJJFRqkSTNiK2pFGmrOIZjVFBcAVm9WFIQhBQ1WTESlhXlebAq//6ozEtCGYhIQ3pXcsUkkpPXJHlfwTZXFif9kZHpc9KbHKZidZQiKYb4i8s54XSABVfksXAeXaeNYudxfLOz5+GJ5/vsl+k2/+5dfr7wflZB9fMmS9BUmFevXUN/eekqsDKa5c9+RyenGT3QYHzmE59yPhuTHYl3y14kis3LNaWkNmc5ZyIrDlT0GoienpYztMGI/GZAm5Xl6mSPhgW34Ckq1Qn3DsclCA3VIZP/H6NY2JHmnmlDRDnS92z84T0RVZ1BTHJFN2NcA5mWXsvlifHpgSGTPhg1KQGqUdTWpM/0iloosNhzImPqeQUiWXqGJ6vaHxFhMujvWeeeV8jtVJNhxvHTDEOHy8nruCil8M15cwHjQYX4gIt+P23zGh3jdI41ZVAUrCAw2Ag0S5gBI94/yJiHAWC2xyOtXDtK0Yxt+JEacsivTMfjYjIRPxuFA7Pawkt8YKWgQ4fMqLMEtwIMUZ/BngMgurDHz8oJ2C/odBb4BrDbDtVG38aHWpAhC4fMIKuBR9Y061f/gSFFHkfu25TTfYHiZ/9wPD3dsRu9ca9WGD+k2ORmAgNHjNuCPfBvnh3smPGuRPRDansuPeJvfjvXM/anCf6FrbJfu20e9c5Ftj7x432J+JqCPfOvnB3sk3o94HIjv27bM/2jd7O/FT9t3d7VV3t3UaN4Dd0Y6A1wPfnrDzPOEuul8W3etI+X8ucJ7bQH+dp/fFMR6m+Mc8qOlcoC0X2PtNzvOej36dgTvsbWHf+0XOGzS40iimtmqjjRKIQRB+VknRPDnWhMgYp9IZA8xMKf1FlVqa6k1yBbp+q9TWdjSjXP4w44vGT91AOqZ5utjsPF2+qhpnkMuQbtu8fUmAdZ9uA2sPvto0W7FKyolij/X3AW8Prfm8bRKCpNvOYv/qIOPsepAZtebSzWdSQS71vgsjbU1mGz5q81WIA7EQbslYxclyI1vUE5HONToCsjIYnBekzF3TznBnaPa0+Qwj923SDjar11RF38bLPu/sDw==</diagram></mxfile>
```

Comment:
You should attach files that are checkable to a pull request - Add a png of your diagram